src/components/donation/Steps/DonateMethods/Stripe/Form.tsx

```diff
@@ -0,0 +1,119 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import { FormProvider, useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import { ObjectSchema, number, object } from "yup";
+import { Props } from "./types";
+import { SchemaShape } from "schemas/types";
+import { useCreateStripePaymentIntentMutation } from "services/apes";
+import { useErrorContext } from "contexts/ErrorContext";
+import ExtLink from "components/ExtLink";
+import LoadText from "components/LoadText";
+import Split from "components/Split";
+import { Field } from "components/form";
+import { appRoutes } from "constants/routes";
+import { TERMS_OF_USE_DONOR } from "constants/urls";
+import AdvancedOptions from "../../../AdvancedOptions";
+
+type FormValues = {
+  amount: number;
+  pctLiquidSplit: number;
+};
+
+export default function Form({
+  advanceOptDisplay,
+  onClientSecretLoaded,
+  widgetConfig,
+  state: {
+    recipient: { id: endowId },
+  },
+}: Props & { onClientSecretLoaded: (clientSecret: string) => void }) {
+  const methods = useForm<FormValues>({
+    resolver: yupResolver(schema),
+    defaultValues: { pctLiquidSplit: 50 },
+  });
+  const { handleError } = useErrorContext();
+  const [createPaymentIntent, { isLoading }] =
+    useCreateStripePaymentIntentMutation();
+
+  const isInsideWidget = widgetConfig !== null;
+
+  const submit = methods.handleSubmit(async (fv) => {
+    try {
+      const { clientSecret } = await createPaymentIntent({
+        amountInCents: fv.amount * 100, // convert to cents
+        endowmentId: endowId,
+        liquidSplitPct: fv.pctLiquidSplit.toString(),
+      }).unwrap();
+      onClientSecretLoaded(clientSecret);
+    } catch (err) {
+      handleError(err, "Failed to load payment platform");
+    }
+  });
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={submit} className="grid gap-4">
+        <Field<FormValues>
+          name="amount"
+          label="Donation amount"
```

Comment:
should add hint that this is in `usd`

Reply:
Fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2598/commits/c27ebc5e18130c772c49717638166f6d817cbe83